JBIDE-20216 add wikitext asciidoc editor; exc...
...lude creole and commonmark plugins (for now)

nickboldt opened the pull request.

mickaelistria commented on jbdevstudio/multiple/jbdevstudio-multiple.target:
Do we specifically need an alternative version for this feature? If not, I believe it would be better to stick to the version which is in Mylyn release or Simrel repositories.
@maxandersen Does your asciidoc extension require a newer version of the wikitext feature?

maxandersen commented on jbdevstudio/multiple/jbdevstudio-multiple.target:
No, it should be using just public API available in Eclipse Mars GA.

nickboldt commented:
Why wouldn't you want to pull the 2.6.0.Nightly so that it's 100% compatible w/ the extras? Why even risk a mix of 2.5 and 2.6 ? Anyway, I merged this as is because of Max's +1 on the jira. So if you want to switch back to using 2.5.R instead of 2.6.N, please reopen the JIRA.

mickaelistria commented:
For JBDS, it's up to us to decide; but for JBT, since we know our users start from an Eclipse bundle, it's often more reliable to depend on the same artifacts as release train that to start using alternative versions that might cause conflicts during installation.

maxandersen commented:
why would we want to use anything but what is in Mars GA ? its just the asciidoctor bundles that should be needed.

maxandersen commented:
also the jira did not list those base ones as being updated so i'm a bit confused what you guys are referring to ;)

nickboldt commented:
Removed: <unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="2.5.0.v20150522-1430"/> 

Added:

 <repository location="http://download.jboss.org/jbosstools/updates/requirements/mylyn-extras/2.6.0.N20150706-2134/"/>
 <!-- JBIDE-20216 wikitext asciidoc editor; exclude creole and commonmark plugins -->
 <unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="2.6.0.N20150706-2134"/>
 <unit id="org.eclipse.mylyn.wikitext.asciidoc.core" version="2.6.0.N20150625-2041"/>
 <unit id="org.eclipse.mylyn.wikitext.asciidoc.ui" version="2.6.0.N20150625-2041"/>
 </location>

Pretty straightforward, no? General rule of thumb is that we bump all the related stuff at the same time, so since the mylyn.extras feature depends on the wikitext_feature, I figured it was best to use 2.6.0.N instead of 2.5.0. 

If you don't agree, please re-open https://issues.jboss.org/browse/JBIDE-20216

maxandersen commented:
that is not the general rule. we prefer stable binaries where possible and shouldn't be taking more things from a nightly site than absolute necessary.

maxandersen commented:
jira reopened.

nickboldt commented:
https://issues.jboss.org/browse/JBIDE-20391 was closed wontfix so closing this too.
